### ISSUE TITLE
ra-data-graphql-simple: Missing paging variables on GET_MANY_REFERENCE

### ISSUE THREAD

**Issue by lnxmad** [marmelab/react-admin#3397]
**What you were expecting:**
Pagination params included in graphQL variables on GET_MANY_REFERENCE call
`page`, `perPage`

**What happened instead:**
Only sending in `filter` variable

**Steps to reproduce:**
Paginate a `ReferenceManyField`

**More Info**
I suspect it needs to mapped in here
https://github.com/marmelab/react-admin/blob/65aea8edb5ed49f13a467eebb9f4709933860cc1/packages/ra-data-graphql-simple/src/buildVariables.js#L248

**Environment**
* React-admin version: 2.9.3
* Last version that did not exhibit the issue (if applicable):
* React version: 16.4.1
* Browser: Chrome

**Comment by fzaninotto** [comment]
Thanks for the report, you're right about the bug and the fix!

**Comment by m4theushw** [comment]
I created a PR to fix this. The `ReferenceManyField` on demo is not working fine with the GraphQL data provider. To be able to test I had to change the `target` prop from `foobar_id` to `foobar.id` since the `buildVariables` picks the first part of the target and adds an `Id` suffix. I also had to change the stubs in the `data-generator` to include a field matching the generated one. I don't know if this is a bug because docs says to use `foobar_id` as target.
